astropy/coordinates/tests/test_frames_with_velocity.py

```diff
@@ -46,23 +46,18 @@ def test_api():
     "kwargs",
     [
         POSITION_ON_SKY,
-        # In Python 3.9 we could write `POSITION_ON_SKY | DISTANCE`
-        {**POSITION_ON_SKY, **DISTANCE},
-        {**POSITION_ON_SKY, **PROPER_MOTION},
-        {**POSITION_ON_SKY, **DISTANCE, **PROPER_MOTION},
-        {**POSITION_ON_SKY, **RADIAL_VELOCITY},
-        {**POSITION_ON_SKY, **DISTANCE, **RADIAL_VELOCITY},
-        {**POSITION_ON_SKY, **PROPER_MOTION, **RADIAL_VELOCITY},
-        {**POSITION_ON_SKY, **DISTANCE, **PROPER_MOTION, **RADIAL_VELOCITY},
+        POSITION_ON_SKY | DISTANCE,
+        POSITION_ON_SKY | PROPER_MOTION,
+        POSITION_ON_SKY | DISTANCE | PROPER_MOTION,
+        POSITION_ON_SKY | RADIAL_VELOCITY,
+        POSITION_ON_SKY | DISTANCE | RADIAL_VELOCITY,
+        POSITION_ON_SKY | PROPER_MOTION | RADIAL_VELOCITY,
+        POSITION_ON_SKY | DISTANCE | PROPER_MOTION | RADIAL_VELOCITY,
         # Now test other representation/differential types:
         CARTESIAN_POSITION,
-        {**CARTESIAN_POSITION, **CARTESIAN_REPRESENTATION_KEYWORD_STR},
-        {**CARTESIAN_POSITION, **CARTESIAN_VELOCITY},
-        {
-            **CARTESIAN_POSITION,
-            **CARTESIAN_VELOCITY,
-            **CARTESIAN_DIFFERENTIAL_KEYWORD_STR,
-        },
```

Comment:
I suspect there might be a few more places where using the dictionary merge operator could lead to fewer lines of code that are not marked with a easily greppable comment, but looking for them would be beyond the scope of this pull request and there's no urgent need to do that anyways.

Reply:
I wish pyupgrade would catch these. Feels deterministic.